raphtory-graphql/src/model/graph/graph.rs

```diff
@@ -374,12 +374,28 @@ impl GqlGraph {
     }
 
     /// Gets (optionally a subset of) the nodes in the graph.
-    async fn nodes(&self, ids: Option<Vec<String>>) -> GqlNodes {
-        let nodes = self.graph.nodes();
-        match ids {
-            None => GqlNodes::new(nodes),
-            Some(ids) => GqlNodes::new(blocking_compute(move || nodes.id_filter(ids)).await),
+    async fn nodes(
+        &self,
+        ids: Option<Vec<String>>,
```

Comment:
`ids` is redundant with the select (also we need to remember to fix the select filter for nodes (right now it applies a subgraph filter which can remove too many nodes if they were only added via edges, can you make sure there is a ticket for this?)